bloom-housing/bloom
# feat: Updates application confirmation numbers

# Pull Request Template

## Issue Overview

This PR addresses #1918


## Description

Updates confirmation numbers in the public and partners app views. To keep backward compatibility, an old number (longer) shows for old applications.

## How Can This Be Tested/Reviewed?
Create a new application, you should see 8-sign code as a confirmation in:

public:
- account: applications list
- confirmation screen
- confirmation email
- csv export

partners:
- Applications list
- Application detail (header and the first section)
- Application edit (header)
- Flags

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
